galileo/opt/include/galileo/opt/TrajectoryOpt.h

```diff
@@ -555,86 +616,55 @@ namespace galileo
         }
 
         template <class ProblemData, class MODE_T>
-        void TrajectoryOpt<ProblemData, MODE_T>::getSolution(solution_t &result)
+        void TrajectoryOpt<ProblemData, MODE_T>::getSolutionSegments(std::vector<solution::solution_segment_data_t> &result, Eigen::VectorXd &times)
```

Comment:
yeah, it doesn't look like you modify times. Const may be best

Reply:
Fixed!